scripts/importPathFormatter.js

```diff
@@ -0,0 +1 @@
+module.exports = path => path.replace(/\.styl$/, '.css')
```

Comment:
missing m ?

Reply:
The multiline flag? Why? It should get only a pathname string, shoudn't it?

Reply:
Sorry, there must a bug in GitHub.

<img width="138" alt="image" src="https://user-images.githubusercontent.com/465582/49650734-8d8b7780-fa2d-11e8-8b04-d3c8c1812a14.png">

Reply:
Oh :o